docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to contribute
+
+Welcome to the contributions page!
+Please read through the page to start your first PR. 🚀
+
+## Steps
+
+Here are the steps to follow for your contribution!
+
+1. fork the repo under your GitHub account
+2. work on an issue, bounty or any other form of a contribution under your fork
+3. format your code using `cargo fmt`
+4. build and test your code locally with `cargo build` and `cargo test --all`
+5. create a pull request merging from your fork into our repo
+
+## Pull request format
+
+When creating a PR, please make sure to describe what the PR is about and provide us with any relevant information.
+Also make sure that *any of* the following questions are answered in the PR description:
+
+- what feature is the code adding?
```

Comment:
We have a good pull_request template file in Plug which we could re-use some points here e.g. bumping runtime version etc.